client/src/views/Report/HorizontalBar.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+    BarChart,
+    Bar,
+    XAxis,
+    YAxis,
+    LabelList,
+    Cell,
+    Tooltip,
+    ResponsiveContainer,
+} from 'recharts';
+
+export default function HorizontalBar(props) {
+    const {
+        data,
+        colorScheme,
+    } = props;
+    return (
+        <ResponsiveContainer
+            height={350}
+            width="100%"
+        >
+            <BarChart
+                data={data}
+                layout="vertical"
+            >
+                <XAxis
+                    type="number"
+                    hide
+                />
+                <YAxis
+                    type="category"
+                    dataKey="name"
+                />
+                <Tooltip />
+                <Bar dataKey="value">
+                    {
```

Comment:
Prefer not adding a newline immediately after the opening braces in this scenario